.coveragerc

```diff
@@ -1,3 +1,39 @@
 [run]
 include =
-  pycobertura/*
\ No newline at end of file
+  pycobertura/*
+omit=
+  **/__pycache__
+  **/dummy*
+  */templates
+  build
+  docs/_build
+  *.egg
+  .tox
+  *.venv
+  tests/dummy.source1/test_dummy.py
+  tests/dummy.source2/test_dummy.py
+  tests/dummy.uncovered/test_dummy.py
+  tests/dummy.uncovered.addcode/test_dummy.py
+  tests/dummy.zeroexit1/test_dummy.py
+  tests/dummy.zeroexit2/test_dummy.py 
+
+[report]
+include =
+  pycobertura/*
+omit=
```

Comment:
What are we trying to do with omit? Any chance we can reduce that list by using a glob pattern that could capture our needs better? Seems like we'd have to remember to update that list when we add new code in the tests/ directory.

Reply:
addressed in 31ec507d42d3362c4fc6e2597fde07703869b4dc.

Reply:
I think even d287593e18f5902945dee841d71561106d4732e8 should work.